src/py42/clients/_archive_access.py

```diff
@@ -288,16 +302,16 @@ def _is_job_complete(self, job_id):
     def _start_restore(self, file_selections):
         num_files = sum([fs.num_files for fs in file_selections])
         num_dirs = sum([fs.num_dirs for fs in file_selections])
-        size = sum([fs.size for fs in file_selections])
-        zip_result = _check_for_multiple_files(file_selections) or None
+        num_bytes = sum([fs.num_bytes for fs in file_selections])
         return self._storage_archive_service.start_restore(
-            guid=self._device_guid,
+            device_guid=self._device_guid,
             web_restore_session_id=self._archive_session_id,
-            path_set=[fs.path_set for fs in file_selections],
+            restore_groups=[
+                {u"backupSetId": -1, u"files": [f.path_set for f in file_selections]}
```

Comment:
I checked my own account but it looks like all my archives are v1. Making sure this behaves as expected with a v3 archive is one thing i'd like to verify if possible @timabrmsn do you have a v3 archive or now how we could test one?

Reply:
> I checked my own account but it looks like all my archives are v1. Making sure this behaves as expected with a v3 archive is one thing i'd like to verify if possible @timabrmsn do you have a v3 archive or now how we could test one?

I talked to Tim Friday, I don't this will work
for v3. We need to figure out if we need to support
restoring from
multiple backup sets or v3 at all. Also, question:  either way, right now we don't
support v3 archives, right? If so, these changes don't contribute to that lack necessarily. But now would
be a good time
to make that work or at least plan for it